src/main/java/EtfRecommendService/etf/EtfQueryRepository.java

```diff
@@ -45,6 +45,25 @@ public List<EtfReturnDto> findEtfsByPeriod(
                 .fetch();
     }
 
+    //페이징 없는 전체조회
+    public List<EtfReturnDto> findEtfsByKeyword(
+            Theme theme,
+            String keyword
+    ) {
+
+        return jpaQueryFactory
+                .select(Projections.constructor(
+                        EtfReturnDto.class,
+                        etfProjection.id,
+                        etfProjection.etfName,
+                        etfProjection.etfCode,
+                        etfProjection.theme
+                ))
+                .from(etfProjection)
```

Comment:
Repository에서 API 응답에 사용되는 DTO(`EtfReturnDto`)를 그대로 가져다 사용하는 것은 별로 좋지 않아요.
여기서는 그냥 엔티티를 return해도 될 것 같네요.
```suggestion
                .selectFrom(etfProjection)
```